.github/workflows/docker-dev-release.yml

```diff
@@ -0,0 +1,131 @@
+name: Development Docker Build
+
+on:
+  schedule:
+    - cron: '15 0 * * *'
```

Comment:
Why do we start all our cron jobs at the same time ? :smile: (I could be wrong but I think others have the exact same timestamp)

Reply:
they do not actually.